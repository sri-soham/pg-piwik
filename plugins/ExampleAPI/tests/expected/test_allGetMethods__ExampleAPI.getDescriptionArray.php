a:5:{i:0;a:1:{i:0;s:5:"piwik";}i:1;a:1:{i:0;s:11:"open source";}i:2;a:1:{i:0;s:13:"web analytics";}i:3;a:1:{i:0;s:4:"free";}i:4;a:1:{i:0;s:45:"Strong message: Свободный Тибет";}}