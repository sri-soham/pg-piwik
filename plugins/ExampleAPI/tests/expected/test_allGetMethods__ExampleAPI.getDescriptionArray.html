<table id="ExampleAPI_getDescriptionArray" border="1">
<thead>
	<tr>
		<th>value</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>piwik</td>
	</tr>
	<tr>
		<td>open source</td>
	</tr>
	<tr>
		<td>web analytics</td>
	</tr>
	<tr>
		<td>free</td>
	</tr>
	<tr>
		<td>Strong message: Свободный Тибет</td>
	</tr>
</tbody>
</table>
